src/Controller/BackendController.php

```diff
@@ -59,6 +59,10 @@ public function loginAction(): Response
     {
         $this->container->get('contao.framework')->initialize();
 
+        if ($this->container->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_FULLY')) {
+            return new RedirectResponse($this->container->get('security.logout_url_generator')->getLogoutUrl());
```

Comment:
Log out the user if fully authenticated?

Reply:
Yes. It was previously the case that when you accessed the login URL while already logged in, the user was logged out. I don't think we should change that as some people might have used that behavior to log a user out.

Reply:
We have changed this in Contao 4 though and we should not revert it now. 😄

Reply:
No we havent changed that, I explicitly tested it on demo.contao.org and you‘re logged out of the backend if you access the login screen.

Reply:
Ok, but now that we have a `/contao/logout` route, this is no longer necessary. I can change it in another PR though.